blog.jh

```diff
@@ -1,9 +1,24 @@
+application {
+  config {
+    baseName blog,
+    applicationType monolith,
+    packageName com.jhipster.demo.blog,
+    prodDatabaseType mysql,
+    cacheProvider hazelcast,
+    buildTool maven,
+    clientFramework react,
+    useSass true,
+    testFrameworks [protractor]
+  }
+  entities *
+}
+
 entity Blog {
 	name String required minlength(3),
 	handle String required minlength(2)
 }
 
-entity Entry {
+entity BlogEntry {
```

Comment:
I'm not sure why you changed this. I've had issues in the past where JHipster said `Entry` was a reserved word. This would probably make more sense as `Post` since that's what we often call them.